src/main/scala/mesosphere/marathon/core/CoreModule.scala

```diff
@@ -21,17 +22,18 @@ import mesosphere.marathon.core.task.tracker.TaskTrackerModule
   * (as long as we have them).
   */
 trait CoreModule {
```

Comment:
This looks like a bigger change, but I just sorted the attributes alphabetically and added `healthModule`.